module_utils/k8s/common.py

```diff
@@ -55,49 +42,24 @@ class OpenShiftObjectHelper(object):
 
 def remove_secret_data(obj_dict):
     """ Remove any sensitive data from a K8s dict"""
-    if obj_dict.get('data'):
-        # Secret data
-        obj_dict.pop('data')
-    if obj_dict.get('string_data'):
-        # The API should not return sting_data in Secrets, but just in case
-        obj_dict.pop('string_data')
-    if obj_dict['metadata'].get('annotations'):
-        # Remove things like 'openshift.io/token-secret' from metadata
-        for key in [k for k in obj_dict['metadata']['annotations'] if 'secret' in k]:
-            obj_dict['metadata']['annotations'].pop(key)
-
-
-def to_snake(name):
-    """ Convert a string from camel to snake """
-    if not name:
-        return name
-
-    def _replace(m):
-        m = m.group(0)
-        return m[0] + '_' + m[1:]
-
-    p = r'[a-z][A-Z]|' \
-        r'[A-Z]{2}[a-z]'
-    return re.sub(p, _replace, name).lower()
-
-
-class DateTimeEncoder(json.JSONEncoder):
-    # When using json.dumps() with K8s object, pass cls=DateTimeEncoder to handle any datetime objects
-    def default(self, o):
-        if isinstance(o, datetime):
-            return o.isoformat()
-        return json.JSONEncoder.default(self, o)
-
-
-class KubernetesAnsibleModuleHelper(AnsibleMixin, KubernetesObjectHelper):
+    #TODO: Discuss this, these are potentially useful things.
```

Comment:
I agree. When I'm looking up a secret, it is most likely because I want the data.

Reply:
Yeah, better I think would be to somehow add these to no_log, don't know if that's possible.